src/main/java/mate/academy/lib/Injector.java

```diff
@@ -57,7 +57,7 @@ public Object getInstance(Class<?> certainInterface) {
         return newInstanceOfClass;
     }
```

Comment:
This method is public and static, but it's not used outside of this class. Consider changing its access modifier to private.